apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -175,6 +176,40 @@ export const DEFAULT_BLOCKS: Record<
     title: "Rick Astley - Never Gonna Give You Up",
     url: "https://www.youtube.com/embed/dQw4w9WgXcQ?si=ggGGn4uvFWAIelWD",
   },
+  // TODO: Replace with actual working API endpoint
+  [DYNAMIC_STATISTICS_TYPE]: {
+    type: `${DYNAMIC_STATISTICS_TYPE}`,
+    apiEndpoint: "https://jsonplaceholder.com/muis_prayers_time",
+    title: "hijri_date",
+    statistics: [
+      {
+        label: "Subuh",
+        key: "subuh",
+      },
+      {
+        label: "Syuruk",
+        key: "syuruk",
+      },
+      {
+        label: "Zohor",
+        key: "zohor",
+      },
+      {
+        label: "Asar",
+        key: "asar",
+      },
+      {
+        label: "Maghrib",
+        key: "maghrib",
+      },
+      {
+        label: "Ishak",
+        key: "isyak",
+      },
+    ],
+    url: "https://www.youtube.com/watch?v=dQw4w9WgXcQ",
```

Comment:
what's this url for?

Reply:
there will be a "view all dates" CTA under the actual date
<img width="650" alt="image" src="https://github.com/user-attachments/assets/f54c5c70-7c81-4054-a7dd-d72c8cac74e6">

reason being people might want to know prayers time of other dates
(similar to https://www.muis.gov.sg/Media/Islamic-Calendar)

discussed with @sehyunidaaa and we will be doing a similar approach where it's a link to a page with the PDFs

Reply:
the label for the CTA is described here:
https://github.com/opengovsg/isomer/blob/709c2511a485b12cc24fbc2cfd09b6b9f7b48b50/apps/studio/src/components/PageEditor/constants.ts#L211